### PR TITLE
chore: use logger function instead of console.log() #3792

### LINKED ISSUE #3792: Update all logging to use new logging method
### Description

We have `console.log` or `console.error` in many places, but now we have a logging function, we should use that and replace every `console.*`

The function can be found in this file `config/logger.js` , and an example of it's usage can be found in `https://github.com/EddieHubCommunity/LinkFree/blob/main/pages/api/users/%5Busername%5D/index.js#L25`

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by sansarmaske**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4837"></a>

### CODE REVIEW

**Review by Ananya2001-an — APPROVED** [review]
Looks good.

File: components/ClipboardCopy.js [hunk 1]
```diff
@@ -20,7 +21,7 @@ const ClipboardCopy = ({ children }) => {
         setIsCopied(false);
       }, 1500);
     } catch (error) {
-      console.log(err);
+      logger.error(err);
```
Comment (eddiejaoude) [inline components/ClipboardCopy.js:24]:
```suggestion
      logger.error(error);
```

**Comment by sansarmaske** [comment]
Sorry missed out few things. I will make the changes. Thanks for the review.

**Comment by sansarmaske** [comment]
I have updated the code with correct syntax for logger.error()

File: components/ClipboardCopy.js [hunk 2]
```diff
@@ -20,7 +21,7 @@ const ClipboardCopy = ({ children }) => {
         setIsCopied(false);
       }, 1500);
     } catch (error) {
-      console.log(err);
+      logger.error(error);
```
Comment (eddiejaoude) [inline components/ClipboardCopy.js:24]: Thank you for fixing 👍 I know this was broken before your changes

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Pull request merged**